cumulus/polkadot-parachain/Cargo.toml

```diff
@@ -10,9 +10,13 @@ license = "Apache-2.0"
 [lints]
 workspace = true
 
+[lib]
+path = "src/lib.rs"
+
 [[bin]]
 name = "polkadot-parachain"
 path = "src/main.rs"
+required-features = ["binary"]
```

Comment:
IMO this is a bad idea. This will require anyone to always pass this feature. Just put the `lib` into a different crate, maybe `polkadot-parachain-omni` or whatever (actually not a good proposal).

Reply:
Done. PTAL !

Reply:
I named it `polkadot-parachain-lib`, but I don't have strong preferences related to that.